easypermissions/src/main/java/pub/devrel/easypermissions/AppSettingsDialogHolderActivity.java

```diff
@@ -0,0 +1,40 @@
+package pub.devrel.easypermissions;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.RestrictTo;
+import android.support.v7.app.AppCompatActivity;
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+public class AppSettingsDialogHolderActivity extends AppCompatActivity implements DialogInterface.OnClickListener {
```

Comment:
Cool! Also, I don't think there's another way to do it without an activity because we need to forward the result to `onActivityResult`.